roles/common/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+
+- name: Log the OS name, version and release
```

Comment:
Well, you can sorta make it work but I don't know if it's any better.

output
```
TASK: [testnode | Print distro info] ******************************************
<magna005.ceph.redhat.com> REMOTE_MODULE setup filter='ansible_distribution_*'
ok: [magna005.ceph.redhat.com]

TASK: [testnode | debug msg="{{info.ansible_facts}}"] *************************
ok: [magna005.ceph.redhat.com] => {
    "msg": "{u'ansible_distribution_version': u'14.04', u'ansible_distribution_release': u'trusty', u'ansible_distribution_major_version': u'14'}"
}
```
task
```
  8 - name: Print distro info
  9   setup:
 10       filter: "ansible_distribution_*"
 11   register: info
 12   tags:
 13      - debug
 14
 15 - debug: msg="{{info.ansible_facts}}"
 16   tags:
 17       - debug
```